Packs/iDefense/Integrations/FeediDefense/FeediDefense_description.md

```diff
@@ -0,0 +1,34 @@
+Fetch indicators from a JSON feed. The integration supports a large amount of user configurations to support different types of JSON feeds.
```

Comment:
You didn't updated it to match your integration.
The description is not accurate, and there are parameters missing